src/detours.cpp

```diff
@@ -76,7 +76,10 @@ namespace rhost {
                 hWndTitleMap.find(hWnd) == hWndTitleMap.end() && 
                 SUCCEEDED(StringCchLengthA(lpWindowName, STRSAFE_MAX_CCH, &len)) && len > 0) {
 
-                hWndTitleMap[hWnd] = std::string(lpWindowName);
+                std::string windowTitle = std::string(lpWindowName);
+                std::wstring_convert<std::codecvt_utf8<char>, char> strConverter;
```

Comment:
This converts from UTF-8 to wchar_t. But the input in this case is not UTF-8.

Since this code is heavily Win32-dependent in any case, and input comes from Win32 also, I think it's easier to just use Win32 functions for this.